dashboard/test/factories.rb

```diff
@@ -553,17 +553,78 @@
     self.end {start + 6.hours}
   end
 
-  factory :school_info do
-    school_type {SchoolInfo::SCHOOL_TYPE_PUBLIC}
-    state {'WA'}
+  # school info
+
+  # this is the only factory used for testing the deprecated data formats (without country).
+  factory :school_info_without_country, class: SchoolInfo do
+    school_type SchoolInfo::SCHOOL_TYPE_PUBLIC
+    state 'WA'
     association :school_district
   end
```

Comment:
Is there a reason why factory inheritance wasn't used here? Anyone reason why I shouldn't tackle doing so?